src/uct/ib/base/ib_iface.h

```diff
@@ -195,6 +195,7 @@ struct uct_ib_iface {
         uint8_t             enable_res_domain;   /* Disable multiple resource domains */
         uint8_t             qp_type;
         size_t              max_iov;             /* Maximum buffers in IOV array */
+        uint8_t             has_gid;             /* Force use GRH in address */
```

Comment:
i meant vice versa..
this parameter says whether we are forced to use global addr, old name was ok
I mentioned `iface->is_global_addr`, which is more like `has_gid` now..

Reply:
rolled back naming
renaming ```iface->is_global_addr``` affects to tons of another stuff which makes this simple PR too huge